lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -442,18 +413,20 @@ def _prepare_environment(self, environment: Environment, log: Logger) -> bool:
                 )
                 break
 
-        if not is_success and all_wait_for_resource:
-            # TODO raise for wait more resource
-            ...
+        if not is_success:
+            if any_wait_for_resource:
+                raise ResourceAwaitableException(
+                    "No available quota, try to deploy later."
```

Comment:
As a result of this exception are we going to retry after some timeout or is this going to fail/skip with this error?